.changeset/fifty-feet-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+"thirdweb": patch
```

Comment:
```suggestion
"thirdweb": minor
```
since we're now changing the params of `thirdweb create` this needs to be a minor version